compiler/Main.hs

```diff
@@ -119,8 +119,11 @@ csCodegen options@Cs {..} = do
             else if fields
                  then PublicFields
                  else Properties
+    constructorMapping = if constructor_parameters
```

Comment:
Nit: `mapping` is not a good term here. In case of fields and types we have them in the schema and we map them in some way to the target language. There are no constructors in the schema so there is nothing to map.